test/fuzz/JsonRpcFuzz2.js

```diff
@@ -0,0 +1,44 @@
+import { createTestProviderTools, providerResultStub } from '../stub/provider';
+import JsonRpcSpec from '@metamask/api-specs';
+
+
+export async function fuzz(data) {
+  const engine  = createTestProviderTools({
+    scaffold: providerResultStub,
+    networkId: 1,
+    chainId: 1,
+    logging: { quiet: true }
+  }).engine
+
+  let request = {
+      id: 1,
+      jsonrpc: '2.0',
+  }
+
+  const promises = []
+
+  JsonRpcSpec.methods.forEach((method) => {
+    request.method = method.name
+    request.params = data.toString()
```

Comment:
params should be an array in most cases (except wallet_watchAsset). and it might have more than one parameter

Reply:
another approach is we could use json schema already built into the api spec (since OpenRPC uses json schema for the types): https://github.com/json-schema-faker/json-schema-faker

this is an example of a naive tool that does this we could adapt for our use case: https://github.com/open-rpc/test-coverage